notebooks/community/vision_ai/video_warehouse_sdk.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<p>Add the template authentication before this cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Hi Andrew,</p><p>About authentication, the reason that I used these gcloud commands is that previously we found a bug b/290266175 (it is not always reproducible) </p><p> </p><p>At that time, I was using </p><p>from google.colab import auth</p><p>auth.authenticate_user(project_id=PROJECT_ID)</p><p> </p><p>Note project_id=PROJECT_ID this part is important for us to use video intelligence api.</p><p> </p><p>But sometimes it fails as reported in b/290266175</p><p> </p><p>So I changed to use </p><p>!gcloud auth login</p><p>!gcloud auth application-default login</p><p> </p><p> </p><p> </p><p> </p>